### DISCUSSION TITLE
How to show selected list items in preview window

### DISCUSSION THREAD

**Discussion by rbusfield** [junegunn/fzf#3573] [Q&A] [answered]
I have a selected list of dozens of items with many different filter string iterations.  In the end, I have many selections.  How can I show the list in the preview window.

**Comment by LangLangBart** [comment, marked as answer]
|     Placeholder      |            Description            |
| :------------------: | :-------------------------------: |
|         `{}`         |           current line            |
|        `{n}`         | current line number (zero index)  |
| `{q}` /`{fzf:query}` |           current query           |
|    `{fzf:prompt}`    |          current prompt           |
|    `{fzf:action}`    | name of the last action performed |

| Placeholder expression flags |                Description                 |
| :--------------------------: | :----------------------------------------: |
|             `+`              | space-separated list of the selected lines |
|             `f`              |               temporary file               |
|             `s`              |            preserve whitespace             |

Try using placeholders with the `--preview` flag.
In your example, displaying all selected items from your list in the preview should work with the following flag:

```bash
# Run the command 'select' some items and notice the preview will show one selection per line.
# '{+f}' gives you the path to the temporary file where your selected lines are stored, thus the 'cat' command.
seq 12 | fzf --preview 'cat {+f}' --multi
```

<img src="https://github.com/junegunn/fzf/assets/92653266/e4e52adc-7868-4869-ae5e-9e9b416be710" width="600">

**Comment by rbusfield** [comment]
Answered in https://github.com/junegunn/fzf/discussions/3574
